From 5ef1e81c38ddf0d0616ad997ac36ceb13893f15f Mon Sep 17 00:00:00 2001
From: MkKvcs <148012377+MkKvcs@users.noreply.github.com>
Date: Wed, 24 Jul 2024 03:12:05 +0200
Subject: [PATCH] hwdb: add axis range corrections for the Lenovo Thinkpad E16

(cherry picked from commit af5195f7a60a06af4135d5b75baae3935252abcd)

Related: RHEL-5950
---
 hwdb.d/60-evdev.hwdb | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hwdb.d/60-evdev.hwdb b/hwdb.d/60-evdev.hwdb
index 3174e19014..0b70a82093 100644
--- a/hwdb.d/60-evdev.hwdb
+++ b/hwdb.d/60-evdev.hwdb
@@ -623,6 +623,13 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO:*pvrThinkPadL14Gen1**
  EVDEV_ABS_35=::44
  EVDEV_ABS_36=::50
 
+# Lenovo Thinkpad E16 Gen1 (Intel)
+evdev:name:SYNA801A:00 06CB:CEC6 Touchpad:dmi:*svnLENOVO:*pvrThinkPadE16Gen1**
+ EVDEV_ABS_00=::11
+ EVDEV_ABS_01=::11
+ EVDEV_ABS_35=::11
+ EVDEV_ABS_36=::11
+
 # Lenovo T460
 evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pn*ThinkPad*T460:*
  EVDEV_ABS_00=1266:5677:44
